src/output.rs

```diff
@@ -94,15 +93,16 @@ fn print_entry_colorized(
     }
 
     if wants_to_quit.load(Ordering::Relaxed) {
+        stdout.flush()?;
```

Comment:
If `flush()` fails, we probably still want to exit.  Actually I think it would be good to check `wants_to_quit` before writing anything instead of after. 

We could even do it outside of `print_entry_colorized()` now, but that'd be for another PR.